EasyPost.Tests/HttpTests/RequestTest.cs

```diff
@@ -17,5 +18,37 @@ public void TestRequestDisposal()
         CustomAssertions.DoesNotThrow(() => request.Dispose());
     }
 
+    [Fact]
+    public void TestQueryParameterList()
+    {
+        const string key = "key";
+        var value = new List<string>
+        {
+            "value1",
+            "value2"
+        };
+        const string key2 = "key2";
+        const string value2 = "value3";
+        var parameters = new Dictionary<string, object>
+        {
+            { key, value },
+            { key2, value2 }
+        };
+
+        Request request = new("https://google.com", "not_a_real_endpoint", Method.Get, ApiVersion.V2, parameters, new Dictionary<string, string> { { "header_key", "header_value" } });
```

Comment:
suggestion: we should be using `example.com` for these kinds of things to be nice to our google neighbors.

Reply:
These requests aren't actually executed, it's just testing the construction process.

Reply:
Even still, best practice